browser/about_flags.cc

```diff
@@ -491,6 +491,9 @@
           FEATURE_VALUE_TYPE(                                                  \
               ntp_background_images::features::kBraveNTPBrandedWallpaperDemo), \
       },                                                                       \
+      {"brave-ntp-search-widget", "Brave Search Widget on the NTP",            \
+       "Enables searching directly from the New Tab Page", kOsDesktop,         \
+       FEATURE_VALUE_TYPE(features::kBraveNtpSearchWidget)},                   \
```

Comment:
trailing comma is missing after `FEATURE_VALUE_TYPE(features::kBraveNtpSearchWidget)`